### PR TITLE
Fix/undefined is not json serializable

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: front/lib/extract_events.ts [hunk 1]
```diff
@@ -154,7 +154,7 @@ async function _processExtractEventsForMarker({
     auth,
     content: contentToProcess,
     marker: marker,
-    schema: { ...schema, description: schema.description ?? undefined },
+    schema: { ...schema, description: schema.description },
```
Comment (spolu) [inline front/lib/extract_events.ts:157]: just schema no?

File: front/types/chat.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export type ChatSessionType = {
   created: number;
   sId: string;
   title: string | null;
-  messages?: ChatMessageType[];
+  messages: ChatMessageType[] | null;
```
Comment (spolu) [inline front/types/chat.ts:39]: I would go for `[]` as null value so `messages: ChatMessageType[]` ?

Comment (spolu) [reply front/types/chat.ts:39]: But I guess our current implemetnation rely on the possibility of null.

**Review by spolu — DISMISSED** [review]
LGTM 

But this need thorough testing.

Will make a pass of testing. Please wait before merging :pray:

File: front/types/chat.ts (hunk 2 above)
Comment (fontanierh) [reply front/types/chat.ts:39]: yes it does, it's risky because in JS `[]` isn't falsy

**Review by spolu — APPROVED** [review]
LGTM

(and tested)

**Pull request merged**